AdminSelfUpgrade.php

```diff
@@ -43,7 +44,7 @@ class AdminSelfUpgrade extends AdminController
     // used for translations
     public static $l_cache;
     // retrocompatibility
-    public $noTabLink = array();
+    public $noTabLink = [];
```

Comment:
I think you can't use this notation with brackets It's available since PHP 5.4 while 1.6 is compatible with older versions So this will prevent using the autoupgrade module when trying to upgrade from 1.6 to an intermediate PrestaShop version (if you don't want to upgrade to the latest version)

Reply:
Last versions of the autoupgrade module is compatible with php >= 5.6 so I guess we're safe here